### ISSUE TITLE
Check that any error messages are helpful

### ISSUE THREAD

**Issue by Angelsaur** [alan-turing-institute/AutSPACEs#55]
### Summary

When a user provides incorrect information in a form, you need to let them know they’ve made a mistake and help them correct that mistake.

### Deadline 

20/02/2021

### What needs to be done?

You can check how well your website is doing this by going to your sample forms and entering some intentionally incorrect or invalid information.

For example, you could enter a future date into a date of birth field or a postcode with too few characters and try submitting the form. This should generate an error message.

Assess whether the error messages make clear what you need to do to correct the mistakes. Messages like ‘invalid date’ or ‘invalid postcode’ are not very helpful: they indicate that the user has made a mistake, but do not explain why the information was wrong and how to correct the error.

Something like ‘You cannot enter a date in the future - please enter a past date’ is much more helpful.

Play around with a few form fields to see how helpful the error messages are.

**Comment by gedankenstuecke** [comment]
Made issue #369 to create an accessibility checklist that can be used for the MVP and going forward. Closing this along with other accessibility checklist items for now.
